src/java/org/apache/cassandra/cql3/CQLStatement.java

```diff
@@ -75,9 +76,9 @@ default Iterable<Function> getFunctions()
      *
      * @param state the current query state
      * @param options options for this query (consistency, variables, pageSize, ...)
-     * @param queryStartNanoTime the timestamp returned by System.nanoTime() when this statement was received
+     * @param requestTime request enqueue / and start times
```

Comment:
nit: Worth mentioning it's dependent on `cql_start_time`?

Reply:
Mentioned this in `baseTimeNanos` if you are OK with this